tests/e2e/util/image/image.go

```diff
@@ -27,10 +27,10 @@ import (
 )
 
 const (
-	TiDBV3Version                 = "v3.0.19"
-	TiDBV3UpgradeVersion          = "v3.0.20"
-	TiDBV4Version                 = "v4.0.9"
-	TiDBV4UpgradeVersion          = "v4.0.10"
+	TiDBV3Prev                    = "v3.0.19"
```

Comment:
Can we remove this version?

Reply:
yes there's no usage for it now...
WDYT @DanielZhangQD

Reply:
fixed in [3ced58a](https://github.com/pingcap/tidb-operator/pull/3770/commits/3ced58a84bb36a0a45043f19337f83722a60cc5f)